pkgs/tools/admin/ejson2env/test-decryption.nix

```diff
@@ -0,0 +1,17 @@
+{ ejson2env, runCommand }:
+runCommand "check-ejson2env.sh" {} ''
```

Comment:
```suggestion
runCommand "check-ejson2env.sh" {
  nativeBuildInputs = [ ejson2env ];
} ''
```

Reply:
Is that a preferred alternative, or an FYI alternative? :)

Reply:
A preferred alternative.

The second argument to `runCommand` declares an environment, similar to like the body of `mkDerivation`.
Then, instead of using things like `${pkg}/bin/prog`, just declare `pkg` as a nativeBuildInput and use  `prog` directly.